### DISCUSSION TITLE
Improving the schema

### DISCUSSION THREAD

**Discussion by bodhish** [Coronasafe/life#184] [Ideas]
Capturing suggestions on unified schema

Current Schema
https://github.com/coronasafe/life/wiki/Life-Data-Structure

**Comment by vpremk** [comment]
For address, Anyway we can make atleast pincode mandatory if its verified ?

**Reply by jitendraag** [reply]
I don't think we can make pincode mandatory. A lot of data from Twitter and other sources doesn't even have an address or name to start with.
